### PR TITLE
[CHORE] Update Person model to support API response

### PR DESCRIPTION
**Pull request by Sharkesm**
- Add `Person:media` property 
- Add `Person:outcome` property 
- Update `Person:media_links` property to `news`

### CODE REVIEW

**Comment by evilpenguin** [comment]
Any reason models have so many optionals now?

**Comment by Sharkesm** [comment]
@evilpenguin Can you pinpoint the ones that shouldn't be`Optional`

File: SayTheirNames/Source/Controller/Home/Person/PersonController.swift [hunk 1]
```diff
@@ -151,11 +151,11 @@ class PersonController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         view.accessibilityIdentifier = "personView"
-        
         self.network.fetchPersonDetails(with: person.identifier) { [weak self] result in
             switch result {
-            case .success(let personResult):
-                self?.configure(with: personResult.person)
+            case .success(let response):
+                print("Response: ", response.person)
```
Comment (bertadevant) [inline SayTheirNames/Source/Controller/Home/Person/PersonController.swift:157]: Comment left here we should delete it

File: SayTheirNames/Source/Model/Media.swift [hunk 2]
```diff
@@ -25,9 +25,11 @@
 import Foundation
 
 public struct Media: Decodable {
-    let url: String
+    let id: Int
+    let personId: Int
```
Comment (evilpenguin) [inline SayTheirNames/Source/Model/Media.swift:29]: This should be `personID` and `imageURL`

File: SayTheirNames/Source/Model/Donation.swift [hunk 3]
```diff
@@ -26,16 +26,20 @@ import UIKit
 
 public struct Donation: Decodable {
     let id: Int
+    let identifier: String
     let title: String
     let description: String
+    let outcome: String?
```
Comment (evilpenguin) [inline SayTheirNames/Source/Model/Donation.swift:32]: Can we make this non-optional?

File: SayTheirNames/Source/Model/Person.swift [hunk 4]
```diff
@@ -82,15 +89,18 @@ struct Person: Decodable {
         self.age = try container.decodeIfPresent(Int.self, forKey: .age)
         self.city = try container.decodeIfPresent(String.self, forKey: .city) ?? ""
         self.country = try container.decodeIfPresent(String.self, forKey: .country) ?? ""
-        self.bio = try container.decodeIfPresent(String.self, forKey: .bio) ?? ""
-        self.context = try container.decodeIfPresent(String.self, forKey: .context) ?? ""
+        self.story = try container.decodeIfPresent(String.self, forKey: .story) ?? ""
+        self.outcome = try container.decodeIfPresent(String.self, forKey: .outcome) ?? ""
+        self.biography = try container.decodeIfPresent(String.self, forKey: .biography) ?? ""
+        self.shareable =  try container.decodeIfPresent(Shareable.self, forKey: .shareable)
```
Comment (evilpenguin) [inline SayTheirNames/Source/Model/Person.swift:95]: We can add `?? Sharable()` here by adding an `init()` method to `Shareable`

File: SayTheirNames/Source/Model/Petition.swift [hunk 5]
```diff
@@ -26,15 +26,19 @@ import Foundation
 
 public struct Petition: Decodable {
     let id: Int
+    let identifier: String
     let title: String
     let description: String
+    let outcome: String?
```
Comment (evilpenguin) [inline SayTheirNames/Source/Model/Petition.swift:32]: Can we make this non-optional?

**Pull request merged**